action.yaml

```
@@ -9,6 +9,10 @@ inputs:
   GITHUB_TOKEN:
     description: The Github token to use for creating the release.
     required: true
+  RELEASE_TYPE:
+    description: The type of release to perform.
```

maybe include the expected values in the description?

```suggestion
    description: The type of release to perform - value should be patch, minor, or major
```

---

good call